tools/setup/emoji/zulip_emoji_map.py

```diff
@@ -0,0 +1,1505 @@
+    '1f4bb': {'canonical_name': 'computer', 'aliases': ['laptop']},
+    '2328': {'canonical_name': 'keyboard', 'aliases': []},
+    '1f5a5': {'canonical_name': 'desktop_computer', 'aliases': []},
+    '1f5a8': {'canonical_name': 'printer', 'aliases': []},
+    # gemoji/unicode is computer_mouse
+    '1f5b1': {'canonical_name': 'computer_mouse', 'aliases': []},
+    '1f5b2': {'canonical_name': 'trackball', 'aliases': []},
+    # arcade seems like a reasonable addition
+    '1f579': {'canonical_name': 'joystick', 'aliases': ['arcade']},
+    # vise seems like a reasonable addition
+    '1f5dc': {'canonical_name': 'compression', 'aliases': ['vise']},
+    # gold record seems more useful, idea came from http://www.11points.com/Web-
+    # Tech/11_Emoji_With_Different_Meanings_Than_You_Think
+    '1f4bd': {'canonical_name': 'gold_record', 'aliases': ['minidisc']},
+    '1f4be': {'canonical_name': 'floppy_disk', 'aliases': []},
+    '1f4bf': {'canonical_name': 'cd', 'aliases': []},
+    '1f4c0': {'canonical_name': 'dvd', 'aliases': []},
+    # videocassette from gemoji/unicode
+    '1f4fc': {'canonical_name': 'vhs', 'aliases': ['videocassette']},
+    '1f4f7': {'canonical_name': 'camera', 'aliases': []},
+    # both of these seem more useful than camera_with_flash
+    '1f4f8': {'canonical_name': 'taking_a_picture', 'aliases': ['say_cheese']},
+    # video_recorder seems like a reasonable addition
+    '1f4f9': {'canonical_name': 'video_camera', 'aliases': ['video_recorder']},
+    '1f3a5': {'canonical_name': 'movie_camera', 'aliases': []},
+    # seems like the best emoji for movie
+    '1f4fd': {'canonical_name': 'projector', 'aliases': ['movie']},
+    '1f39e': {'canonical_name': 'film', 'aliases': []},
+    # both of these seem more useful than telephone_receiver
+    '1f4de': {'canonical_name': 'landline', 'aliases': ['home_phone']},
+    '260e': {'canonical_name': 'phone', 'aliases': ['telephone']},
+    '1f4df': {'canonical_name': 'pager', 'aliases': []},
+    '1f4e0': {'canonical_name': 'fax', 'aliases': []},
+    '1f4fa': {'canonical_name': 'tv', 'aliases': ['television']},
+    '1f4fb': {'canonical_name': 'radio', 'aliases': []},
+    '1f399': {'canonical_name': 'studio_microphone', 'aliases': []},
+    # volume seems more useful
+    '1f39a': {'canonical_name': 'volume', 'aliases': ['level_slider']},
+    '1f39b': {'canonical_name': 'control_knobs', 'aliases': []},
+    '23f1': {'canonical_name': 'stopwatch', 'aliases': []},
+    '23f2': {'canonical_name': 'timer', 'aliases': []},
+    '23f0': {'canonical_name': 'alarm_clock', 'aliases': []},
+    '1f570': {'canonical_name': 'mantelpiece_clock', 'aliases': []},
+    # times_up and time_ticking seem more useful than the hourglass names
+    '231b': {'canonical_name': 'times_up', 'aliases': ['hourglass_done']},
+    # seems like the better hourglass. Also see Objects/36
+    '23f3': {'canonical_name': 'time_ticking', 'aliases': ['hourglass']},
+    '1f4e1': {'canonical_name': 'satellite_ante
```

Comment:
extra space here before the `#`?

Reply:
Sorry didn't get what you meant here?

Reply:
hm, not sure how that comment ended up here. I meant in the last line of the file, there seems to be an extra space before the `#`.

Reply:
If that "last line" refers to an explanation then that should have been fixed in this version as I have updated the related script.

Reply:
looks like it is fixed, maybe I was looking at an old version. :+1: